README.md

```diff
@@ -35,11 +36,14 @@ For Execution role, assign a role that has the following policies:
  * [`AWSLambdaBasicExecutionRole`](https://gist.github.com/bernadinm/6f68bfdd015b3f3e0a17b2f00c9ea3f8#file-all_aws_managed_policies-json-L1447-L1473)
 
 #### Environment Variables
- * `PIPELINE_KEY` (required): Your pipeline http source node key.
- * `PIPELINE_URL` (required): Pipeline Ingestion URL
+ * `INGESTION_KEY` (required): Your ingestion key for Log Analysis or a Pipeline source.
+ * `INGESTION_URL` (required): Ingestion URL for Log Analysis or Pipeline
+ * `PAYLOAD_FORMAT` (required): Choose either `log_analysis` or `pipeline` depending on which product the data is being sent to.
```

Comment:
`depending on your preferred ingestion method.`

But, It doesn't seem like its necessary to make people specify this